backend/stakepoold/rpcclient.go

```diff
@@ -111,66 +109,68 @@ func connectWalletRPC(cfg *config) (*dcrrpcclient.Client, semver, error) {
 }
 
 func walletFetchUserTickets(ctx *appContext) (map[chainhash.Hash]string, string) {
-	// This is suboptimal to copy and needs fixing.
-	users := make(map[string]userdata.UserVotingConfig)
-	ctx.RLock()
-	for k, v := range ctx.userVotingConfig {
-		users[k] = v
-	}
-	ctx.RUnlock()
+	var (
+		ticketcount, usercount int
+	)
 
 	userTickets := make(map[chainhash.Hash]string)
 
+	log.Infof("Running GetTickets, this may take awhile...")
+	tickets, err := ctx.walletConnection.GetTickets(false)
```

Comment:
This takes a long time with 1000s of tickets.  I'm not sure this is practical unless `GetTickets` is accelerated.